<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Shraddha-PC" failures="7" tests="33" name="Test" time="208.361" errors="0" timestamp="2022-06-26T14:36:13 IST">
  <testcase classname="com.listeners.Runner" name="test1" time="0.042"/>
  <testcase classname="com.listeners.Runner" name="test2" time="0.021">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.listeners.Runner.test2(Runner.java:29)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="com.listeners.Runner" name="test3" time="0.005"/>
  <testcase classname="com.AssertJ.AssertDemo" name="customClassTest" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [com.AssertJ.Employee@5d9526eb] but found [com.AssertJ.Employee@6d38e5b2]">
      <![CDATA[java.lang.AssertionError: expected [com.AssertJ.Employee@5d9526eb] but found [com.AssertJ.Employee@6d38e5b2]
at com.AssertJ.AssertDemo.customClassTest(AssertDemo.java:104)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- customClassTest -->
  <testcase classname="com.AssertJ.AssertDemo" name="numbersTest" time="0.598"/>
  <testcase classname="com.AssertJ.AssertDemo" name="stringTest" time="0.045"/>
  <testcase classname="com.AssertJ.AssertJodaDemo" name="dateTimeTest" time="0.47"/>
  <testcase classname="com.AssertJ.AssertJSoftAssertions" name="assertBDDStyleTest" time="0.008"/>
  <testcase classname="com.dataprovider.latest.DataProviderDependencyInjection" name="test1" time="0.004"/>
  <testcase classname="com.dataprovider.latest.DataProviderDependencyInjection" name="test1" time="0.004"/>
  <testcase classname="com.dataprovider.latest.DataProviderTest" name="test1" time="0.005"/>
  <testcase classname="com.dataprovider.latest.DataProviderTest" name="test1" time="0.003"/>
  <testcase classname="com.dataprovider.latest.DataProviderTest2" name="test1" time="0.003"/>
  <testcase classname="com.dataprovider.latest.DataProviderTest2" name="test1" time="0.003"/>
  <testcase classname="com.dataprovider.latest.DataProviderTest2" name="test2" time="0.003"/>
  <testcase classname="com.dataprovider.latest.DataProviderTest2" name="test2" time="0.004"/>
  <testcase classname="com.dataprovider.latest.DataProviderTest2" name="test2" time="0.024"/>
  <testcase classname="com.dataprovider.latest.DataProviderWithExcel" name="test" time="0.002"/>
  <testcase classname="com.dataprovider.latest.DataProviderWithExcel" name="test" time="0.001"/>
  <testcase classname="com.dataprovider.latest.DataProviderWithExcel" name="test" time="0.001"/>
  <testcase classname="com.dataprovider.latest.DataProviderWithJSON" name="testDPWithJson" time="0.004"/>
  <testcase classname="com.dataprovider.latest.DataProviderWithJSON" name="testDPWithJson" time="0.003"/>
  <testcase classname="com.extentReports.ExtentReportsDemo" name="test" time="51.484"/>
  <testcase classname="com.tmb.tests.ExampleTest" name="SeleniumTest2" time="0.003"/>
  <testcase classname="com.tmb.tests.ExampleTest" name="SeleniumTest3" time="0.004"/>
  <testcase classname="com.pageFactory.Runner" name="test1" time="66.484"/>
  <testcase classname="com.tmb.tests.ExampleTest" name="SeleniumTest1" time="54.16">
    <failure type="java.lang.AssertionError" message="Test Failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Test Failed expected [true] but found [false]
at com.tmb.tests.ExampleTest.SeleniumTest1(ExampleTest.java:42)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- SeleniumTest1 -->
  <testcase classname="com.pageFactory.Runner" name="test" time="110.8">
    <failure type="org.openqa.selenium.NoSuchSessionException" message="invalid session id
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;SHRADDHA-PC&amp;apos;, ip: &amp;apos;192.168.29.116&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;15.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 102.0.5005.115, chrome: {chromedriverVersion: 102.0.5005.61 (0e59bcc00cc4..., userDataDir: C:\Users\SHRADD~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:60543}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}">
      <![CDATA[org.openqa.selenium.NoSuchSessionException: invalid session id
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SHRADDHA-PC', ip: '192.168.29.116', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '15.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 102.0.5005.115, chrome: {chromedriverVersion: 102.0.5005.61 (0e59bcc00cc4..., userDataDir: C:\Users\SHRADD~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:60543}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:285)
at org.openqa.selenium.remote.RemoteWebElement.clear(RemoteWebElement.java:124)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at com.sun.proxy.$Proxy20.clear(Unknown Source)
at com.pageFactory.PageFactoryDemo.findElements10Times(PageFactoryDemo.java:24)
at com.pageFactory.Runner.test(Runner.java:30)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase classname="com.pageFactory.Runner" name="test2" time="110.878">
    <failure type="org.openqa.selenium.NoSuchSessionException" message="invalid session id
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;SHRADDHA-PC&amp;apos;, ip: &amp;apos;192.168.29.116&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;15.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 102.0.5005.115, chrome: {chromedriverVersion: 102.0.5005.61 (0e59bcc00cc4..., userDataDir: C:\Users\SHRADD~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:60543}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
*** Element info: {Using=name, value=q}">
      <![CDATA[org.openqa.selenium.NoSuchSessionException: invalid session id
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SHRADDHA-PC', ip: '192.168.29.116', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '15.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 102.0.5005.115, chrome: {chromedriverVersion: 102.0.5005.61 (0e59bcc00cc4..., userDataDir: C:\Users\SHRADD~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:60543}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
*** Element info: {Using=name, value=q}
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByName(RemoteWebDriver.java:404)
at org.openqa.selenium.By$ByName.findElement(By.java:284)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at com.pageFactory.IRecommendNoPageFactoryDemo.findElements10Times(IRecommendNoPageFactoryDemo.java:21)
at com.pageFactory.Runner.test2(Runner.java:43)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="com.tmb.tests.HomepageTests" name="Test2" time="21.338">
    <failure type="java.lang.AssertionError" message="expected [8] but found [13]">
      <![CDATA[java.lang.AssertionError: expected [8] but found [13]
at com.tmb.tests.HomepageTests.Test2(HomepageTests.java:49)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
  <testcase classname="com.tmb.tests.LoginPageTestBasic" name="Test1" time="9.111"/>
  <testcase classname="com.tmb.tests.LoginPageTest" name="Test1" time="5.746"/>
  <testcase classname="com.tmb.tests.LoginPageTestBasic" name="Test2" time="17.293">
    <failure type="org.openqa.selenium.WebDriverException" message="java.net.ConnectException: Failed to connect to localhost/127.0.0.1:6461
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;SHRADDHA-PC&amp;apos;, ip: &amp;apos;192.168.29.116&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;15.0.1&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: java.net.ConnectException: Failed to connect to localhost/127.0.0.1:6461
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SHRADDHA-PC', ip: '192.168.29.116', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '15.0.1'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:92)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:285)
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:106)
at com.tmb.tests.LoginPageTestBasic.Test2(LoginPageTestBasic.java:48)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: java.net.ConnectException: Failed to connect to localhost/127.0.0.1:6461
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:155)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
... 19 more
Caused by: java.net.ConnectException: Connection refused: no further information
at java.base/sun.nio.ch.Net.pollConnect(Native Method)
at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:660)
at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:549)
at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)
at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:333)
at java.base/java.net.Socket.connect(Socket.java:648)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
... 39 more
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- Test -->
